lib/oxrdf/src/dataset.rs

```diff
@@ -925,8 +925,8 @@ impl PartialEq for Dataset {
 impl Eq for Dataset {}
 
 impl<'a> IntoIterator for &'a Dataset {
-    type IntoIter = Iter<'a>;
     type Item = QuadRef<'a>;
+    type IntoIter = Iter<'a>;
```

Comment:
Not sure about this, RustRover prefers to order `type` based on the decaration order whereas [rustfmt `reorder_impl_items` option](https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=#reorder_impl_items) prefers to use alphabetical order. I would tend to follow rustfmt because it is also usable by people not using RustRover but happy to be convinced otherwise. What do you think?

Reply:
I agree that we shouldn't rely on IntelliJ, but it seems that `reorder_impl_items` is still not default, and most users have been asking for "same as in trait" if you look at the first comment in https://github.com/rust-lang/rustfmt/issues/3363#issuecomment-622580136